app/controllers/procedures_controller.rb

```diff
@@ -1,58 +1,59 @@
 class ProceduresController < ApplicationController
-     #Currently shows all Procedures
-    def index
-        @procedures = Procedure.all
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  #Currently shows all pending Procedures
+  def index
+    @procedures = Procedure.where("status = 'approved'")
```

Comment:
use a hash to search: `where(status: 'approved')`

Reply:
Changed to hash in procedures, items and admins controllers.